charts/kibana/Chart.yaml

```diff
@@ -5,7 +5,7 @@ maintainers:
   - name: Wiremind
     url: https://github.com/wiremind/wiremind-helm-charts
 name: kibana
-version: 8.5.1
+version: 8.5.2
```

Comment:
Note: this is so weird because chart version was synchronized with es version, which of course never works... It is ok for us

Reply:
I found it weird but I can't pull if I don't update chart version